packages/dashboard/src/customization/propertiesSections/thresholdSettings/thresholdsSection.tsx

```diff
@@ -89,6 +89,27 @@ const ThresholdsSection: FC<ThresholdsSectionProps> = ({
 
   const [isExpanded, setIsExpanded] = useExpandable();
 
+  /**
+   * Wrapper for updating styled thresholds
+   * If a threshold's comparison operator is "=" then always show a visible line
+   * If the update to thresholds is a new style then pass it into this function, otherwise use the style state
+   */
+  const handleUpdateStyledThresholds = useCallback(
```

Comment:
Why do we use `useCallback` for this function? What would happen if we didn't use `useCallback`?

Reply:
I thought it would help resolve the saved global thresholdStyle, but it wasn't working. Removing it functions the same, so I removed it.